packages/esm-patient-forms-app/src/forms/forms-dashboard.component.tsx

```diff
@@ -21,6 +28,10 @@ const FormsDashboard: React.FC<DefaultWorkspaceProps> = () => {
   const { patient, patientUuid } = usePatient();
   const { data: forms, error, mutateForms } = useForms(patientUuid, undefined, undefined, !isOnline, config.orderBy);
   const { currentVisit } = useVisitOrOfflineVisit(patientUuid);
+  const sessionUser = useSession();
+  const sessionPrivileges = sessionUser?.user?.privileges;
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  let editableForms = [];
```

Comment:
I'd consider doing the filtering inside a `useMemo()` hook, since otherwise, we may re-render the FormsList on every render that hits this.